### ISSUE TITLE
autodiscoverFilter not respected by job scheduler

### ISSUE THREAD

**Issue by mikebryant** [renovatebot/pro#74]
I'm trying to use 0.17.0, against a self-hosted Gitlab instance

It looks like the scheduler doesn't respect the `autodiscoverFilter` setting, though I can see it being recognised:

```
{"name":"renovate","hostname":"pro-1-79b77998df-6bzl2","pid":1,"level":20,"config":{"autodiscover":true, ...... ,"token":"***********","autodiscoverFilter":"includes*/**"},"msg":"Combined config","time":"2019-10-16T16:36:49.780Z","v":0}
```

It then looks at all repositories, whether or not they match the filter

(We were hoping to use this to slowly migrate from our current OSS renovate setup)

**Comment by rarkins** [comment]
I'll see if the autodiscoverFilter functionality can be hooked into the scheduler easily or not

**Comment by rarkins** [comment]
Please see https://github.com/renovatebot/pro/releases/tag/0.18.0-pre.0

**Comment by mikebryant** [comment]
I'm trying 0.18.0-pre.0, but it's still not working for me:
```
... `"autodiscoverFilter":"includes*/**"`

Then it runs for a repository that's called `systems/maven`, for example - doesn't seem to be respecting the filter
```

**Comment by rarkins** [comment]
Looks like I messed up the build :-/

Stay tuned!

**Comment by rarkins** [comment]
@mikebryant sorry about that - I have now published `renovate/pro:0.18.0-pre.1` and yanked the faulty pre.0 tag. I double triple checked it so hopefully it's ok for you!

**Comment by mikebryant** [comment]
Can confirm this is working for me now, thanks!
